easybuild/easyconfigs/s/SuperLU_DIST/SuperLU_DIST-6.4.0-foss-2020a.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = "EB_SuperLU"
+
+name = 'SuperLU_DIST'
+version = '6.4.0'
+
+homepage = 'https://crd-legacy.lbl.gov/~xiaoye/SuperLU/'
+description = """SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems
+ of linear equations on high performance machines."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+github_account = 'xiaoyeli'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ["v%(version)s.tar.gz"]
+checksums = ['cb9c0b2ba4c28e5ed5817718ba19ae1dd63ccd30bc44c8b8252b54f5f04a44cc']
+
+builddependencies = [('CMake', '3.16.4')]
+
+dependencies = [
+    ('ParMETIS', '4.0.3'),
+]
+
+configopts = '-DTPL_PARMETIS_INCLUDE_DIRS="${EBROOTPARMETIS}/include" '
+configopts += '-DTPL_PARMETIS_LIBRARIES="${EBROOTPARMETIS}/lib/libparmetis.a;${EBROOTPARMETIS}/lib/libmetis.a" '
+
+# Some tests run longer than default 1500s timeout on fairly big machine (36 cores).
+# Increasing timeout to 3000 resolves the timeout error.
+# Be ware that tests run ~2hrs.
+pretestopts = 'export ARGS="$ARGS --timeout 3000" && '
```

Comment:
I'd say this is too excessive to run during the installation...

Is there any way to only run a subsuite of the tests?
If not, we should leave this commented out, imagine doing the installation on just 3 cores...

Reply:
Is Patchfile with modifications to Ctest a way?

Reply:
Maybe we can use the `--tests-regex` option to pick a couple of tests, or use `--exclude-regex` to exclude the very long test(s)?